lib/plug/router.ex

```diff
@@ -346,8 +373,15 @@ defmodule Plug.Router do
           {b, options}
         options[:do] ->
           Keyword.pop(options, :do)
+        options[:to] ->
+          {target, opts} = Keyword.pop(options, :to)
+          init_opts = Keyword.get(options, :init_opts, [])
+          b = quote bind_quoted: [target: target, init_opts: init_opts] do
```

Comment:
Can we get rid of the one-letter variable `b`? I assume it means `body`, which would be infinitely more clear if so 😄